yosh/shell.py

```diff
@@ -40,11 +87,53 @@ def execute(cmd_tokens):
         signal.signal(signal.SIGINT, handler_kill)
         # Spawn a child process
         if platform.system() != "Windows":
-            # Unix support
-            p = subprocess.Popen(cmd_tokens)
-            # Parent process read data from child process
-            # and wait for child process to exit
-            p.communicate()
+            found = 0
+            std = 0
+            waitpid = 0
+            # Auto find the command
+            for i in os.getenv("PATH").split(":"):
+                if os.path.exists(i + "/" + cmd_name):
+                    for a, el in enumerate(cmd_tokens):
+                        # Find the '>' and '>>' sign
+                        if el == ">" or el == ">>":
```

Comment:
`>` means to rewrite. 
`>>` means to append.
It should not be handled with same process.